mobile-nav.js

```diff
@@ -0,0 +1,24 @@
+const cancel = document.getElementById('cancel');
+const links = document.querySelectorAll('.link');
+const hamburger = document.getElementById('hamburger');
+
+links.forEach((links) => {
+  links.addEventListener('click', () => {
+    const x = document.getElementById('nav');
+    x.style.display = 'none';
+  });
+});
+
+cancel.addEventListener('click', () => {
+  const x = document.getElementById('nav');
+  x.style.display = 'none';
+});
+
+hamburger.addEventListener('click', () => {
+  const x = document.getElementById('nav');
+  if (x.style.display === 'none') {
+    x.style.display = 'flex';
+  } else {
+    x.style.display = 'none';
+  }
```

Comment:
- [ ] There are some issues in your design. kindly consider fixing them as per requirements.
   - When the user clicks on hamburger-menu, then the hamburger should disappear and the cross should appear in its place. but in your design user can see and access both which is wrong so consider fixing it.
 your design
![re](https://github.com/batoolfatima2135/Portfolio/assets/96683843/3673f9d6-3c35-466b-ab2c-81362dfacb41)


figma design
![qw](https://github.com/batoolfatima2135/Portfolio/assets/96683843/66cecee1-5772-45ca-987d-7f7c2b3a62c7)